src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -294,7 +294,7 @@ public function configureOptions(OptionsResolver $resolver)
 
         $choiceListNormalizer = function (Options $options, $choiceList) use ($choiceListFactory) {
             if ($choiceList) {
-                @trigger_error('The "choice_list" option is deprecated since version 2.7 and will be removed in 3.0. Use "choice_loader" instead.', E_USER_DEPRECATED);
+                @trigger_error(sprintf('The "choice_list" option of a %s form type is deprecated since version 2.7 and will be removed in 3.0. Use "choice_loader" instead.', __CLASS__), E_USER_DEPRECATED);
```

Comment:
I propose to be a bit shorter : `The "choice_list" option of %s is deprecated...`